src/types.ts

```diff
@@ -101,3 +101,32 @@ export interface Column {
   stylingInfo: string[];
   sortDirection: string;
 }
+
+export interface PageInfo {
+  page: number;
+  rowsPerPage: number;
+  rowsPerPageOptions: number[];
+}
+
+export type SetPageInfo = stardust.SetStateFn<{
+  page: number;
+  rowsPerPage: number;
+  rowsPerPageOptions: number[];
+}>;
+
+export type Announce = ((arg0: AnnounceArgs) => void) | undefined;
```

Comment:
this should already exist

Reply:
I tried with `announce: Announce` which does not work. But I gonna look into it again.

Reply:
Fixed